### ISSUE TITLE
[BUG] Boilerplate Cucumber-Selenium producing a chromedriver ENOENT error

### ISSUE THREAD

**Issue by matbgn** [IBMa/equal-access#446]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [x] `boilerplates`
- [ ] `other`

## Detailed description

> Node version: v12.19.0
> Platform: Windows10 Pro

After shallowing the provided boilerplate and installing dependencies (npm i), running test script is producing an error: "chromedriver ENOENT" see additional information for full error report.

## Steps to reproduce the issue

1. git clone --depth 1 https://github.com/IBMa/equal-access.git accessibility-checker
2. cd accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium
3. npm i
4. npm run test
5. error!

## Additional information

- Error Console:
``` node.js
"C:\Program Files (x86)\nodejs\node.exe" "C:\Program Files (x86)\nodejs\node_modules\npm\bin\npm-cli.js" test --scripts-prepend-node-path=auto

> @ test C:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium
> cucumber-js

events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium\node_modules\chromedriver\bin\chromedriver ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn C:\\dev\\workspace\\accessibility-checker\\accessibility-checker\\boilerplates\\cucumber-selenium\\node_modules\\chromedriver\\bin\\chromedriver',
  path: 'C:\\dev\\workspace\\accessibility-checker\\accessibility-checker\\boilerplates\\cucumber-selenium\\node_modules\\chromedriver\\bin\\chromedriver',
  spawnargs: [ '--port=55619' ],
  [Symbol(originalCallSite)]: [ CallSite {}, CallSite {}, CallSite {} ],
  [Symbol(mutatedCallSite)]: [ CallSite {}, CallSite {}, CallSite {} ]
}
npm ERR! Test failed.  See above for more details.
```

- Error Log:
``` node.js
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files (x86)\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'test'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v12.19.0
4 verbose run-script [ 'pretest', 'test', 'posttest' ]
5 info lifecycle @~pretest: @
6 info lifecycle @~test: @
7 verbose lifecycle @~test: unsafe-perm in lifecycle true
8 verbose lifecycle @~test: PATH: C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium\node_modules\.bin;C
:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium\node_modules\.bin;C:\cygwin\bin;C:\cygwin\lib;C:\cygwin\lib\awk;C:\cygwin\usr\sbin;C:\Python3\Scripts\;C:\Python3\;C:\Program Files (x86)\Com
mon Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\
TortoiseSVN\bin;C:\Program Files (x86)\Plantronics\Spokes3G\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Common Files\Adobe\
AGL;C:\Program Files\Git\cmd;C:\Program Files (x86)\nodejs\;C:\ProgramData\chocolatey\bin;C:\HashiCorp\Vagrant\bin;C:\Program Files (x86)\Brackets\command;C:\Program Files\SourceGear\Common\DiffMerge\;C:\Program Files\Docker\Docker\
resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\gettext-iconv\bin;C:\Program Files\Calibre2\;C:\Users\mat\AppData\Local\Microsoft\WindowsApps;C:\Users\mat\AppData\Roaming\npm;C:\Users\mat\AppData\Local\Progra
ms\Microsoft VS Code\bin
9 verbose lifecycle @~test: CWD: C:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium
10 silly lifecycle @~test: Args: [ '/d /s /c', 'cucumber-js' ]
11 silly lifecycle @~test: Returned: code: 1  signal: null
12 info lifecycle @~test: Failed to exec test script
13 verbose stack Error: @ test: `cucumber-js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:314:20)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:314:20)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid @
15 verbose cwd C:\dev\workspace\accessibility-checker\accessibility-checker\boilerplates\cucumber-selenium
16 verbose Windows_NT 10.0.19042
17 verbose argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "test"
18 verbose node v12.19.0
19 verbose npm  v6.14.8
20 error code ELIFECYCLE
21 error errno 1
22 error @ test: `cucumber-js`
22 error Exit status 1
23 error Failed at the @ test script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by tombrunet** [comment]
Node / NPM have known issues with Windows paths that have spaces in them. Please try in a directory without spaces.
